admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js

```diff
@@ -92,8 +92,9 @@ export default class OrderProductRenderer {
     $(OrderViewPageMap.productsPanel).detach().appendTo($modificationPosition);
     $modificationPosition.closest('.row').removeClass('d-none');
 
-    // Show column location
-    this.toggleColumnLocation(OrderViewPageMap.productsCellLocation, true);
+    // Show column location & refunded
+    this.toggleColumn(OrderViewPageMap.productsCellLocation);
```

Comment:
```suggestion
    this.toggleColumn(OrderViewPageMap.productsCellLocation, true);
```
@NeOMakinG are you sure you don't need this parameter to force display here? Same on the line under

Reply:
@atomiix as you worked on this, what do you think ?

Reply:
@jolelievre @NeOMakinG Yes I'm sure, it should not be forced! :)